base/intfuncs.jl

```diff
@@ -937,7 +937,7 @@ Return `true` if and only if the extrema `typemax(T)` and `typemin(T)` are defin
 """
 hastypemax(::Base.BitIntegerType) = true
 hastypemax(::Type{Bool}) = true
-hastypemax(::Type{T}) where {T} = applicable(typemax, T) && applicable(typemin, T)
```

Comment:
Imo just inline the `hasmethod_one_type` here and remove it. Feels excessive to have to look into another file for such a small utility function to understand what this is doing.